tests/e2e/run-ui-e2e.sh

```diff
@@ -32,10 +34,18 @@ run_ui_e2e_tests() {
 
     if [[ "${LOAD_BALANCER}" == "lb" ]]; then
         local hostname
-        hostname=$("$ROOT/tests/e2e/get_hostname.py" "${API_HOSTNAME}")
+        if [[ "${API_HOSTNAME}" =~ ^[0-9]+\.[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
+            info "Getting hostname from IP: ${API_HOSTNAME}"
+            hostname=$("$ROOT/tests/e2e/get_hostname.py" "${API_HOSTNAME}")
+        else
+            hostname="${API_HOSTNAME}"
+        fi
+        info "Hostname for central-lb alias: ${hostname}"
         echo "central-lb ${hostname}" > /tmp/hostaliases
         export HOSTALIASES=/tmp/hostaliases
         export UI_BASE_URL="https://central-lb:443"
+    elif [[ "${LOAD_BALANCER}" == "route" ]]; then
+        die "unsupported LOAD_BALANCER ${LOAD_BALANCER}"
```

Comment:
I don't know enough about this environment to be sure, but this looks like it this aborts if we are in an OpenShift environment.

I'm going to assume that's not all that's going on, and not block the PR on this. I would like to know why this is here.

Reply:
We unfortunately do not cover OpenShift routes in CI (the option is there in the deploy scripts to use it but it is not done (added https://issues.redhat.com/browse/ROX-15022)).
I added this check in case someone tried to sneak it in without properly adding support . The same logic for 'lb' would _probably_ work but falling through to 'localhost' would falsely work.